src/core/LoadingSpinner/LoadingSpinner.md

```diff
@@ -0,0 +1,95 @@
+```js
+import { LoadingSpinner } from 'suomifi-ui-components';
+
+<>
+  <div aria-live="assertive">
+    <LoadingSpinner
+      status="loading"
+      variant="normal"
+      textAlign="right"
+      text="Loading"
+    />
+    <br />
+    <LoadingSpinner status="success" text="Loading finished" />
+    <br />
+    <LoadingSpinner status="failed" text="Loading failed" />
+    <br />
+    <LoadingSpinner
+      status="loading"
+      variant="small"
+      textVisibility="hidden"
+      text="Loading"
+    />
+    <br />
+    <LoadingSpinner
+      status="success"
+      variant="small"
+      text="Loading finished"
+    />
+    <br />
+    <LoadingSpinner
+      status="failed"
+      variant="small"
+      textAlign="right"
+      text="Loading failed"
+    />
+  </div>
+</>;
+```
+
+### LoadingSpinner in action
+
+```js
+import { useState } from 'react';
+
+import {
+  LoadingSpinner,
+  VisuallyHidden
+} from 'suomifi-ui-components';
+
+const [visible, setVisible] = useState(false);
+const [loaded, setLoaded] = useState(0);
+const [status, setStatus] = useState('loading');
+
+const timeout = () => {
```

Comment:
Could this be named something a bit more descriptive? Not a big deal, but caught my eye.

Reply:
Yes it could, but I have no good ideas. startTimer ?

Reply:
Yeah, it's not the easiest to name... Maybe `runLoader` or something along those lines?

Reply:
That sounds good. Changed it now.